Formula/mp3val.rb

```diff
@@ -18,12 +18,13 @@ class Mp3val < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:    "b36cb11d26af2abdb0e0d811bb91f24d9b7e78bfdd8cd65f0aa2283c08725feb"
   end
 
-  # Apply this upstream commit to fix build on Linux:
-  # https://sourceforge.net/p/mp3val/subversion/95/
-  # Remove with next release.
-  patch :DATA
-
   def install
+    # Apply this upstream commit to fix build on Linux:
+    # https://sourceforge.net/p/mp3val/subversion/95/
+    # Remove with next release.
+    inreplace "crossapi.cpp",
+              "od=open(szNewName,O_WRONLY|O_CREAT|O_TRUNC);",
+              "od=open(szNewName,O_WRONLY|O_CREAT|O_TRUNC, S_IRUSR|S_IWUSR);"
```

Comment:
Is this the same patch as before?

Also, this is patch only for linux build?

Reply:
Yes, I am replacing the same line as in the initial patch.
The initial patch was also applied on macOS and seems to do no harm (note that upstream did not make the change Linux specific)

Reply:
I see, 👍